src/Converters/Converters.Tests/ValueConverterGroupTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Globalization;
+using FormsCommunityToolkit.Converters;
+using NUnit.Framework;
+using Xamarin.Forms;
+
+namespace Converters.Tests
+{
+    [TestFixture]
+    public class ValueConverterGroupTests
+    {
+        class MockConverter : IValueConverter
```

Comment:
I would prefer to see this type in a separate file (maybe even folder/namespace) so it can be reused for other tests.

Reply:
done